modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -598,6 +606,32 @@
 				this.selectedSource = this.textSources[0];
 			}
 		},
+		selectDefaultIosTrack: function (defaultLangKey) {
+			var _this = this;
```

Comment:
@GuyLivni all native HTML5 logic should be in the native.js implementation.
And in general FYI - you can use `this.embedPlayer.getPlayerElement()` instead of `_this.embedPlayer.getVideoHolder().find("video")[0]`